.github/workflows/qa-ec2.yml

```diff
@@ -50,8 +72,6 @@ jobs:
           chmod 0400 vault-secret.txt
           echo "${{ secrets.ROOT_CA }}" > ./ops/ansible/aws/rootCA.pem
           chmod 0400 ops/ansible/aws/rootCA.pem
-          echo "${{ secrets.ANSIBLE_VARS }}" > ./ops/ansible/aws/vars.yml
-          aws s3 cp ./ops/ansible/aws/vars.yml s3://nexodus-io/ec2-e2e/
```

Comment:
This line is gone now. Does it matter?

Reply:
@russellb I should have mentioned it in the PR, its not required anymore. Ty!